mypy is unable to find stub files

Steps to reproduce.

1. Create new folder, initialize and activate a python3.7 virtual environment there.
```bash
mkdir check_strenum && cd !$
python3.7 -m venv env && . env/bin/activate
```

2. Install strenum and mypy
```bash
pip install strenum mypy
```

3. Create a test python script, name it check_strenum.py:
```python
from strenum import StrEnum

class MyCheck(StrEnum):
    PASS: str = 'pass'
    FAIL: str = 'fail'
```

4. Check code with `mypy`:
```bash
mypy --install-types --strict check_strenum.py
```

Actual result:
```
check_strenum.py:1: error: Skipping analyzing "strenum": module is installed, but missing library stubs or py.typed marker  [import]
check_strenum.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
check_strenum.py:3: error: Class cannot subclass "StrEnum" (has type "Any")  [misc]
Found 2 errors in 1 file (checked 1 source file)
```

Expected result:
```
Success: no issues found in 1 source file
```

Here are the contents of the package after it's installed with pip.
```
$ ll env/lib/python3.7/site-packages/strenum/
total 36K
drwxr-xr-x  3 eugenesqr eugenesqr 4.0K Apr 27 13:16 .
drwxr-xr-x 19 eugenesqr eugenesqr 4.0K Apr 27 13:16 ..
drwxr-xr-x  2 eugenesqr eugenesqr 4.0K Apr 27 13:16 __pycache__
-rw-r--r--  1 eugenesqr eugenesqr 8.4K Apr 27 13:16 __init__.py
-rw-r--r--  1 eugenesqr eugenesqr 2.0K Apr 27 13:16 mixins.py
-rw-r--r--  1 eugenesqr eugenesqr 3.4K Apr 27 13:16 _name_mangler.py
-rw-r--r--  1 eugenesqr eugenesqr  498 Apr 27 13:16 _version.py
```
Note that `*.pyi` files are missing.

If we manually copy stub files from the repo along with `py.typed`, it starts working:
```bash
cd env/lib/python3.7/site-packages/strenum/
touch py.typed
curl -O https://raw.githubusercontent.com/irgeek/StrEnum/master/strenum/__init__.pyi
```

I believe these files should be included into the result package. For example via `setup.py`:
```python
package_data={'strenum': ['*.pyi', 'py.typed']},
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,4 @@
-*.pyi
-*.typed
+include *.pyi
+include *.typed
 include versioneer.py
 include strenum/_version.py

